.github/workflows/test-macos.yaml

```diff
@@ -41,4 +42,6 @@ jobs:
           rubygems: 3.3.26
           bundler: 2.3.26 # needed to fix issue with steep on Ruby 3.0/3.1
       - run: bundle install
+      - if: ${{ matrix.ruby == '3.4' }}
+        run: gem update --system 3.5.21
```

Comment:
The latest version of `rubygem` is 3.5.22. However, version 3.5.22 changes macos files ([see here](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3522--2024-10-16)) and causes the rspec tests to fail. Specifically, `/spec/datadog/tracing/contrib_spec.rb:58` is unable to find the correct file paths because the root gem path has changed.

I'm unsure that hard-coding rubygem v3.5.21 is the best solution, so very open to any suggestions!

Reply:
I would suggest adding a note to this effect to the source for future readers.